### ISSUE TITLE
Implement passkey authentication

### ISSUE THREAD

**Issue by valentinoeval** [BookStackApp/BookStack#4686]
### Describe the feature you'd like

Hey ! Thank's a lot for your great and hard work !
Now, authentication with login/password and with 2FA is cumbersome ..
We can make it easier to login with the new passkey technique.
Do you think this is a feature you could implement in the future ?

Source : https://blog.joe.codes/implementing-passkey-authentication-in-your-laravel-app

### Describe the benefits this would bring to existing BookStack users

Improve security and facilitate the user login process

### Can the goal of this request already be achieved via other means?

Yes, with authentication with 2FA but longer

### Have you searched for an existing open/closed issue?

- [X] I have searched for existing issues and none cover my fundamental request

### How long have you been using BookStack?

3 months to 1 year

**Comment by ssddanbrown** [comment]
Thanks for the request @valentinoeval.
Somewhat related to #3912.

**Comment by kayvanaarssen** [comment]
@ssddanbrown Any updates on this?

**Comment by ssddanbrown** [comment]
@kayvanaarssen Nope. There's little demand, not really enough to justify this as an extra auth mechanism, plus I'd still like to see the real world usage of passkeys to understand patterns/usage/UX-requirements/potential-issues.

**Comment by ssddanbrown** [comment]
BookStack is now [managed on Codeberg](https://codeberg.org/bookstack/bookstack).

As part of this change, we're closing all existing GitHub issues and pull requests.
You'll instead find this issue on Codeberg here: 
https://codeberg.org/bookstack/bookstack/issues/4686
  
Any further discussion for this will now be done on Codeberg.
Therefore this GitHub thread will be locked.
